apps/api/src/applicant/ienapplicant.controller.ts

```diff
@@ -43,6 +47,12 @@ export class IENApplicantController {
   })
   @UseInterceptors(ClassSerializerInterceptor)
   @ApiResponse({ status: HttpStatus.OK, type: EmptyResponse })
+  @RouteAcceptsRoles(
+    ValidRoles.HEALTH_AUTHORITY,
+    ValidRoles.HEALTH_MATCH,
+    ValidRoles.MINISTRY_OF_HEALTH,
+  )
+  @UseGuards(AuthGuard)
```

Comment:
can we move @UserGuards to the controller level?
like this
@Controller('ien')
@UseGuards(AuthGuard)

and last 2 function in the same file missing @RouteAcceptRoles.

Reply:
This is a good idea, I will make the changes